Objects/call.c

```diff
@@ -2,6 +2,24 @@
 #include "frameobject.h"
 
 
+int
+_PyObject_HasFastCall(PyObject *callable)
+{
+    if (PyFunction_Check(callable)) {
+        return 1;
+    }
+    else if (PyCFunction_Check(callable)) {
+        PyMethodDef *method = ((PyCFunctionObject*)callable)->m_ml;
+        int flags = method->ml_flags & ~(METH_CLASS | METH_STATIC | METH_COEXIST);
```

Comment:
"I think clearing these flags is not needed. It would be better to use PyCFunction_GET_FLAGS()."

Right, done. But I prefer explicit if().